packages/protocol/migrations_ts/3_1_score_manager.ts

```diff
@@ -0,0 +1,17 @@
+import { SOLIDITY_08_PACKAGE } from '@celo/protocol/contractPackages'
```

Comment:
Update filename to `31_score_manager.ts`

Reply:
renamed to 26_101 - 26_103 to have it more consistent with what is already there